webapp/WEB-INF/templates/admin/admin_login.html

```diff
@@ -49,4 +49,23 @@
   var bgImg = 'url(' + randomImages[rndNum] + ')';
   $("#login-page").css('background-image', bgImg );
 });
+$(document).ready(function() {
+	var buttonGroup = $("<span class='input-group-btn'><button type='button' class='btn btn-flat' title='${i18n('portal.admin.admin_login.showPassword')?html}'><span class='fa fa-eye'></span><span class='sr-only'>${i18n('portal.admin.admin_login.showPassword')?html}</span></button></span>");

```

Comment:
Should we use "message?js_string?html here" ? This way if we add the ' character it won't break ?

Reply:
It seems so, yes (for the occurence inside the `title` attibute)

Reply:
why not all 5 occurences ?

Reply:
it cannot hurt to do it everywhere, I guess, but only this occurence is susceptible to break if a `'` is unescaped.